### DISCUSSION TITLE
Is there any place where I can get api documentation?

### DISCUSSION THREAD

**Discussion by kartik-579** [aquasecurity/trivy#3553] [Q&A]
I am wanting to integrate trivy in our open source project and thus need api references.

**Comment by itaysk** [comment]
Hi, Trivy is a CLI utility, so you generally just execute it with certain flags, and consume the result which can be in json format for integration. If you're looking for something else, please clarify.
BTW after you've integrated Trivy into your project, feel free to mention it in the Trivy Ecosystem page https://aquasecurity.github.io/trivy/v0.37/ecosystem/

**Reply by kartik-579** [reply]
Actually I am looking for integration via REST or RPC (not preferring to execute commands in my codebase). I can try importing trivy asa library and use internal methods directly but that thing would not be maintainable. Please let me know what you think of this scenario.

**Reply by itaysk** [reply]
You can use Trivy's code directly. There's no REST/RPC API. If you're interested, Trivy Premium (paid commercial product) has it.
